packages/astro/CHANGELOG.md

```diff
@@ -1,5 +1,137 @@
 # astro
 
+## 5.0.0-alpha.1
+
+### Major Changes
+
+- [#11798](https://github.com/withastro/astro/pull/11798) [`e9e2139`](https://github.com/withastro/astro/commit/e9e2139bf788893566f5a3fe58daf1d24076f018) Thanks [@matthewp](https://github.com/matthewp)! - Unflag globalRoutePriority
+
+  The previously experimental feature `globalRoutePriority` is now the default in Astro 5.
```

Comment:
```suggestion
  The previously [experimental feature `globalRoutePriority`](https://docs.astro.build/en/reference/configuration-reference/#experimentalglobalroutepriority) is now the default in Astro 5.
``` 
You can ignore this, but it makes me feel better to have suggested it @matthewp !  :joy: